inlang/source-code/sdk/src/loadProject.ts

```diff
@@ -80,13 +82,47 @@ export const loadProject = async (args: {
 		// -- settings ------------------------------------------------------------
 
 		const [settings, _setSettings] = createSignal<ProjectSettings>()
-		createEffect(() => {
+		createEffect(async () => {
+			let projectId: string | undefined
+
+			try {
+				projectId = await nodeishFs.readFile(projectPath + "/projectId", {
+					encoding: "utf-8",
+				})
```

Comment:
Why not `/project-id.txt` it's a txt file right?

Reply:
its not a real text file. usually fiels that only have an id dont have an ending, see pid files etc.